integration-tests/src/test/resources/domaininimagewdt.yaml

```diff
@@ -7,6 +7,7 @@ configuredManagedServerCount: 4
 initialManagedServerReplicas: 2
 exposeAdminT3Channel: true
 exposeAdminNodePort: true
+voyagerWebPort: 30333
```

Comment:
This can be changed to loadBalancerWebPort, it should work as the code looks for loadBalancerWebPort if voyagerWebPort is not set. Eventually we should remove voyagerWebPort and use only loadBalancerWebPort, so minimizing the use of voyagerWebPort attribute is better.

Reply:
Did some experiment and research. For TRAEFIK since there is only one  loadbalancer instance for multiple ingress resources (for multiple backends/WLS domains) we can't not use multiple webports for TRAEFIK. So under existing integ test framework we need to maintain a separate voyagerWebPort